node/src/main/scala/coop/rchain/node/configuration/commandline/Options.scala

```diff
@@ -490,6 +490,17 @@ final case class Options(arguments: Seq[String]) extends ScallopConf(arguments)
   addSubcommand(dataAtName)
   addSubcommand(contAtName)
 
+  val findDeploy = new Subcommand("find-deploy") {
+    descr(
+      "Searches for a block containing the deploy with provided id."
+    )
+    val deployId = opt[String](
```

Comment:
...or maybe you should parse the deployId here using the coop.rchain.node.configuration.commandline.Base16Converter

Reply:
Done